docker/airflow.Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN apt-get update -yqq \
     && add-apt-repository ppa:ubuntugis/ppa \
     && apt-get install gdal-bin -yqq \
     && apt-get install libgdal-dev -yqq \
+    && apt-get install rclone \
```

Comment:
I now think that this dockerfile is meant for building the local image, while the kubernetes one is for GCP, that's why they're so similar. So I'm not sure rclone will be needed for the local image.

Reply:
True, but I think it is worth including it here, to save frustration with some future debugging scenario.